src/Altinn.Platform/Altinn.Platform.Register/Register/Program.cs

```diff
@@ -1,164 +1,286 @@
+    services.Configure<PlatformSettings>(config.GetSection("PlatformSettings"));
+
+    services.AddSingleton<IAuthorizationHandler, AccessTokenHandler>();
+    services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+    services.AddSingleton<ISigningKeysResolver, SigningKeysResolver>();
+
+    services.AddAuthentication(JwtCookieDefaults.AuthenticationScheme)
+          .AddJwtCookie(JwtCookieDefaults.AuthenticationScheme, options =>
+          {
+              GeneralSettings generalSettings = config.GetSection("GeneralSettings").Get<GeneralSettings>();
+              options.JwtCookieName = generalSettings.JwtCookieName;
+              options.MetadataAddress = generalSettings.OpenIdWellKnownEndpoint;
+              options.TokenValidationParameters = new TokenValidationParameters
+              {
+                  ValidateIssuerSigningKey = true,
+                  ValidateIssuer = false,
+                  ValidateAudience = false,
+                  RequireExpirationTime = true,
+                  ValidateLifetime = true,
+                  ClockSkew = TimeSpan.Zero
+              };
+
+              if (builder.Environment.IsDevelopment())
+              {
+                  options.RequireHttpsMetadata = false;
+              }
+          });
+
+    services.AddAuthorization(options =>
+    {
+        options.AddPolicy("PlatformAccess", policy => policy.Requirements.Add(new AccessTokenRequirement()));
+    });
+
+    services.AddHttpClient<IOrganizations, OrganizationsWrapper>();
+    services.AddHttpClient<IParties, PartiesWrapper>();
+    services.AddHttpClient<IAuthorization, AuthorizationWrapper>();
+
+    if (!string.IsNullOrEmpty(applicationInsightsKey))
+    {
+        services.AddSingleton(typeof(ITelemetryChannel), new ServerTelemetryChannel() { StorageFolder = "/tmp/logtelemetry" });
+        services.AddApplicationInsightsTelemetry(applicationInsightsKey);
+        services.AddApplicationInsightsTelemetryProcessor<HealthTelemetryFilter>();
+        services.AddApplicationInsightsTelemetryProcessor<IdentityTelemetryFilter>();
+        services.AddSingleton<ITelemetryInitializer, CustomTelemetryInitializer>();
+
+        logger.LogInformation($"Startup // ApplicationInsightsTelemetryKey = {applicationInsightsKey}");
+    }
+
+    // Add Swagger support (Swashbuckle)
+    services.AddSwaggerGen(c =>
+    {
+        c.SwaggerDoc("v1", new OpenApiInfo { Title = "Altinn Platform Register", Version = "v1" });
+        IncludeXmlComments(c);
+    });
+}
+
+void IncludeXmlComments(SwaggerGenOptions swaggerGenOptions)
+{
+    try
+    {
+        string xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+        string xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+        swaggerGenOptions.IncludeXmlComments(xmlPath);
+    }
+    catch (Exception e)
+    {
+        logger.LogWarning("Prorgam // Exception when attempting to include the XML comments file(s): " + e.Message);
```

Comment:
Typo in the message. Include the exception as first parameter in the method call. Consider removing the string concatination. The exception message will be included as a part of the exception.